index.php

```diff
@@ -24,16 +24,16 @@
     <a href="index.php"><h1 class="text-5xl">Blog</h1></a>
     <?php $session = new SessionHandles();
         if (!$session->checkUserLoggedIn()) {
-            echo '<div class="flex gap-5">
-            <a href="login.php">Login</a>
-            </div>
-            <div class="flex gap-5">
-            <a href="registration.php">Register</a>
-            </div>';
-        } else {
-            echo '<div class="flex gap-5">
-            <a href="logout.php">Logout</a>
-            </div>'; } ?>
+        echo '<div class="flex gap-5">
+        <a href="login.php">Login</a>
+        <a href="registration.php">Register</a>
+        </div>'; } else {
+        echo '<div class="flex gap-5">
+        <a href="addPost.php">Create Post</a>
```

Comment:
The create post link is in the wrong place, it should b einside the same div as logout/login/register